packages/input-edit/src/modules/inputEditTable/reducer.js

```diff
@@ -0,0 +1,38 @@
+import {reducer as reducerUtil} from 'tocco-util'
+
+import * as actions from './actions'
+
+const initialState = {
+  data: {},
+  inputEditForm: [],
+  inputDataForm: {},
+  sorting: {}
+}
+
+const setValue = (state, action) => {
+  const {inputDataKey, node, value} = action.payload
+  const indexToUpdate = Object.keys(state.data).find(index => state.data[index].pk === inputDataKey)
+  return {
+    ...state,
+    data: {
+      ...state.data,
+      [indexToUpdate]: {
```

Comment:
wouldnt it be more practical to use the pk as index?

Reply:
Good point. I used to do it like that, but then changed it around when implementing the arrow key navigation, because it made that way easier. But in the end I changed another thing that does not need these indexes anymore. So now this can be switched back as well =)